config/application.rb

```diff
@@ -13,6 +13,9 @@
 require "sprockets/railtie"
 # require "rails/test_unit/railtie"
 
+# Silence all Ruby 2.7 deprecation warnings in development mode
+$VERBOSE = nil if Rails.env.development?
```

Comment:
Does this line work when you put it in `development.rb` before the `Rails.application.configure` block?

Reply:
Yes!

I added it to `development.rb` and `test.rb` (disabling it for CI by default) too

Reply:
Neat, I like having it in the specific environemnts as compared to `application.rb`. Thanks!